rev_news/drafts/edition-15.md

```diff
@@ -27,52 +27,52 @@ Git Merge conference on April 5 2016.
 
 At the Git Contributor Summit on April 4th, just after the lunch
 break, the state of the participation of the Git project in the Google
-Summer of Code (GSoC) and Outreachy was briefly mentioned, then Jeff
-King, alias Peff, talked about the state of the Git project as part of
+Summer of Code (GSoC) and Outreachy were briefly mentioned. Jeff King,
+alias Peff, then talked about the state of the Git project as part of
 the [Software Freedom Conservancy (SFC)](https://sfconservancy.org/).
 
 Git is one of the SFC Member Projects. The SFC manages the legal and
-administrative aspects of the Git project. The SFC also manages Git's
+administrative aspects of the Git project, and also manages Git's
 money, which amounts to around $20000. This money comes mostly from
-donations made on git-scm.org, GSoC mentor stipends and book
+donations made on git-scm.org, GSoC mentor stipends, and book
 royalties. The money has been used in the past to pay for some
-developers to travel to conferences like Git Merge. It may be used in
+developers to travel to conferences like Git Merge, and may be used in
 the future to pay Outreachy student stipends.
 
 The governance of the Git project as part of the SFC is in the hands
-of Junio Hamano, Shawn Pearce and Jeff King. The main activity of the
+of Junio Hamano, Shawn Pearce, and Jeff King. The main activity of the
 governance consists in handling trademark issues related to the "Git"
 trademark that the project owns.
 
-There are a number of projects who call themselves Git XXX and the
+There are a number of projects who call themselves "Git XXX" and the
 trademark policy on git-scm.org is used to decide if those trademarks
 are approved or not.
 
-Another activity is related to defending the license which is the
-GPLv2. There have been some vendors who distributed Git with some
-changes, but did not provide the source code of the Git version they
-distributed. For now it has been possible to resolve these cases, but
-it is not completely clear if some vendors currently fullfill all
-their obligations. In case some developers who contributed to Git
-wants to take a close look at what the vendors are doing, the SFC can
-help them.
+Another activity is related to defending the license, which is the
+GPLv2. For example, there have previously been vendors distributing
+Git with some changes, but without providing the source code for the Git version they
+were distributing. So far it has been possible to resolve these cases, but
+it is not completely clear if all vendors are currently fullfilling all of
+their obligations. If any developers who have contributed to Git
+want to take a closer look at what the vendors are doing, the SFC are
```

Comment:
I think "is" is better in this context, as it is talking about the group a a whole (see http://public.wsu.edu/~brians/errors/group.html, for example). I don't think either is egregiously wrong, though.

BTW, not a new problem in this PR, but a while ago Conservancy sent out some "style points" for referring to it (them? :wink:):

> - please do not add a "the" in front of "Software Freedom Conservancy" or "Conservancy" in any written context.  We've admittedly used a "the" intermittently in the past, but we'd like to be consistent going forward.  So, instead of "Project X is a member of the Conservancy," just say "Project X is a member of Conservancy."
>
> - please use "Conservancy" as the abbreviation for "Software Freedom Conservancy."  We're discontinuing the use of the acronym "SFC," and we ask your help in doing the same.

I don't think it's worth trying to change existing references, but perhaps something to keep in mind for future writing.